test/SparkLendMainnetIntegration.t.sol

```diff
@@ -0,0 +1,287 @@
+// SPDX-License-Identifier: AGPL-3.0
+pragma solidity ^0.8.0;
+
+import "forge-std/Test.sol";
+
+import { RateSourceMock } from "./mocks/RateSourceMock.sol";
+
+import { IAaveOracle }                  from "aave-v3-core/contracts/interfaces/IAaveOracle.sol";
+import { IPoolAddressesProvider }       from "aave-v3-core/contracts/interfaces/IPoolAddressesProvider.sol";
+import { IPool }                        from "aave-v3-core/contracts/interfaces/IPool.sol";
+import { IPoolConfigurator }            from "aave-v3-core/contracts/interfaces/IPoolConfigurator.sol";
+import { IDefaultInterestRateStrategy } from "aave-v3-core/contracts/interfaces/IDefaultInterestRateStrategy.sol";
+import { IERC20 }                       from "erc20-helpers/interfaces/IERC20.sol";
+import { SafeERC20 }                    from "erc20-helpers/SafeERC20.sol";
+
+import { FixedPriceOracle }                   from "../src/FixedPriceOracle.sol";
+import { CappedOracle }                       from "../src/CappedOracle.sol";
+import { PotRateSource }                      from "../src/PotRateSource.sol";
+import { RateTargetBaseInterestRateStrategy } from "../src/RateTargetBaseInterestRateStrategy.sol";
+import { RateTargetKinkInterestRateStrategy } from "../src/RateTargetKinkInterestRateStrategy.sol";
+
```

Comment:
```suggestion
import { FixedPriceOracle }                   from "src/FixedPriceOracle.sol";
import { CappedOracle }                       from "src/CappedOracle.sol";
import { PotRateSource }                      from "src/PotRateSource.sol";
import { RateTargetBaseInterestRateStrategy } from "src/RateTargetBaseInterestRateStrategy.sol";
import { RateTargetKinkInterestRateStrategy } from "src/RateTargetKinkInterestRateStrategy.sol";
```

Reply:
fixed and ya I like this better ill stop using relative paths